setup.cfg

```diff
@@ -0,0 +1,63 @@
+[metadata]
```

Comment:
I'm curious of your thoughts re using setup.cfg vs moving to just using `.toml` file instead and using `poetry` as package manager?